documentation/docs/14-types.md

```diff
@@ -3,3 +3,70 @@ title: Types
 ---
 
 **TYPES**
+
+### Generated types
+
+The [`RequestHandler`](#sveltejs-kit-requesthandler) and [`Load`](#sveltejs-kit-load) types both accept a `Params` argument allowing you to type the `params` object:
+
+```diff
+/// file: src/routes/[foo]/[bar]/[baz].js
+-/** @type {import('@sveltejs').RequestHandler} */
```

Comment:
This diff feels a little confusing. Is this a migration guide kind of thing leaking out to somewhere we should be more forward looking?

Reply:
It's meant to be read as 'you _could_ type `params` like this' in order to set up 'but you _should_ do it like this', but will concede it might not be landing. Any thoughts on a better alternative?

Reply:
Maybe just be more explicit about it? Mention the old types inline in the paragraph, and then just write the manually typed params in the code block, which would even let you use JS syntax highlighting and hover types, rather than syntax highlighted as a diff.

Reply:
Oh, I see the link to the types in the diff code block already works in the deployed site, so that part isn't as much of a concern.

What is missing, though, is the path of this file, so the part about renaming the directory just below this is confusing.

Reply:
Tweaked it a bit. The file path is there, though maybe it's liable to get missed?

<img width="723" alt="image" src="https://user-images.githubusercontent.com/1162160/156097513-8a2fbe17-88a4-4eb9-b390-5f3bf1992bf8.png">

Reply:
Made it a bit more obvious 4471a8b2061ab1b27ef6b0f762cb30d68e5289f3

Reply:
It wasn't displaying the filename `src/routes/[foo]/[bar]/[baz].js` at the top of the `diff` code block before

![image](https://user-images.githubusercontent.com/16696352/156171644-a0d67c1f-fe62-4b9d-a90a-dd6d87abcec7.png)

but I do see it there now. I think the new version is good.